package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Templates for articles on both iOS and Android next-gen apps",
   "main": "ArticleTemplates/",
   "scripts": {
-    "setup": "echo ArticleTemplates/assets/css/ >> .git/info/exclude && echo ArticleTemplates/assets/build/ >> .git/info/exclude",
+    "setup": "if [ ! -d .git/info ]; then  mkdir -p .git/info; fi; echo ArticleTemplates/assets/css/ >> .git/info/exclude && echo ArticleTemplates/assets/build/ >> .git/info/exclude",
```

Comment:
Good catch! I didn't think you could clone without it creating a `.git/info` dir :thinking: